### ISSUE TITLE
App Groups not working after deploying the app to Test Flight?

### ISSUE THREAD

**Issue by prashant000chaudhary** [j3k0/cordova-plugin-openwith#36]
It is working fine with the local build but handler function is not getting called in test flight build. It seems because app groups are not working in the deployed build.

**Comment by NiklasMerz** [comment]
Does your project show any issues in Xcode? Did you add the group to your distribution provisioning profile?

**Comment by prashant000chaudhary** [comment]
No, It is working fine when I install the app using Xcode but when I release it to test flight handler function is not getting called and app groups are already added

**Comment by NiklasMerz** [comment]
Did you check your devices logs in Xcode? I had some issues with app extensions that were not correctly build. Please go to View->Devices and check logs and console.

**Comment by AlexanderHieser** [comment]
Maybe this is realted to #48. I already posted a possible solution there.
